tests/test_graphical_units/test_tables.py

```diff
@@ -46,3 +46,679 @@ def test_DecimalTable(scene):
         np.linspace(0, 0.9, 20).reshape(5, 4),
     )
     scene.add(t)
+
+
+##############################################################################
+# tests for add_highlighted_cell(function STARTS HERE
```

Comment:
Is it really necessary to add 68 graphical unit tests for this? The bug you have fixed could practically even be tested without a graphical test by checking that the previously thrown error does not appear any longer.

Graphical tests are fine, but they should test fairly specific things. I am not convinced yet that this is the case here; please explain your motivation for adding so many tests. (Practially, given that so many of these are so similar, some of these grouped tests should probably have been written as parameterized tests.)